src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesAddTraceDrawer/components/CreateTraceFormTraceDefinitionItems/CreateTraceFormTraceDefinitionItems.vue

```diff
@@ -44,37 +44,30 @@ function getFileInputSuccessMessage (file: File | null) {
           <template #default="{ field }">
             <AvFileUpload
               id="trace-file-upload"
-              :model-value="field.state.value?.name"
+              :file-upload-title="t('global.information.fileUpload.title')"
+              :file-upload-description="t('global.information.fileUpload.dragAndDrop')"
+              :delete-button-label="t('global.buttons.delete')"
+              :model-value="field.state.value"
               :accept="acceptedFileTypes"
               :aria-label="`${t('student.views.studentToolsTracesView.studentToolsTracesAddTraceDrawer.createTraceForm.fileUpload.title')} ${t('student.views.studentToolsTracesView.studentToolsTracesAddTraceDrawer.createTraceForm.fileUpload.subtitle')}`"
               :error="field.state.meta.errors.join(', ')"
               :valid-message="getFileInputSuccessMessage(field.state.value)"
               @change="(files) => handleFileChange(files, field.handleChange)"
-            >
-              <div class="file-upload-content">
-                <p class="b2-regular">
-                  {{ t('student.views.studentToolsTracesView.studentToolsTracesAddTraceDrawer.createTraceForm.fileUpload.title') }}
-                </p>
-                <p class="caption-light">
-                  {{ t('student.views.studentToolsTracesView.studentToolsTracesAddTraceDrawer.createTraceForm.fileUpload.subtitle') }}
-                </p>
-              </div>
-              <template #hint>
-                <p
-                  class="caption-light"
-                >
-                  <template
-                    v-for="(item, index) in filesTypesMaxSize"
-                    :key="item.type"
-                  >
-                    {{ $t(item.type) }} : <span class="file-upload-content__hint">{{ item.size }}</span>
-                    <span v-if="index < filesTypesMaxSize.length - 1"> • </span>
-                  </template>
-                </p>
-              </template>
-            </AvFileUpload>
+              @update:model-value="(value) => field.handleChange(value)"
+            />
           </template>
         </form.Field>
+        <div>
+          <p class="caption-light">
+            <template
+              v-for="(item, index) in filesTypesMaxSize"
+              :key="item.type"
+            >
+              {{ $t(item.type) }} : <span class="file-upload-content__hint">{{ item.size }}</span>
```

Comment:
```suggestion
              {{ $t(item.type) }} : <span class="caption-bold">{{ item.size }}</span>
```

Et supprime dans la partie style le style associé à `file-upload-content__hint`